CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ text.
 keepalive events.
 - The sensu-agent binary is now located at ./cmd/sensu-agent.
 - Sensuctl no longer uses auto text wrapping.
+- The backend no longer requires embedded etcd. External etcd instances can be
+used by providing the --no-embed option. In this case, the client will dial
```

Comment:
Very minor: would `--no-embedded-etcd` be clearer and more "future proof" (as in, no-embed could mean many things ultimately)?

Reply:
I changed it to be --no-embed-etcd